sklearn/linear_model/_cd_fast.pyx

```diff
@@ -622,8 +622,8 @@ def enet_coordinate_descent_gram(floating[::1] w,
 
 def enet_coordinate_descent_multi_task(floating[::1, :] W, floating l1_reg,
```

Comment:
@jeremiedbb I already see the syntax `floating[::1, :] W` in master, did I miss something?

Reply:
We can use memoryviews when we are sure that the array has been created in scikit-learn because then we know it's not read-only. When it's a user provided array (X, y), we don't have that guarantee and fused typed memoryviews don't work with read only arrays.